src/test/test_rpc.ts

```diff
@@ -1019,3 +1019,37 @@ export default async function test(client: TestClient) {
 `;
   await new TestCase(schema, handler, tester).run();
 });
+
+test('rpc works for Partial param types', pass, async () => {
```

Comment:
Do we also get a good TypeScript type after going there and back?  (May I hope it's a _reasonable_ type??)

Reply:
It becomes `Partial_of_User_end` which is acceptable.
We're going to have to rewrite concord at some point and avoid the typescript->json->typescript conversion.